stacks/core/prod/traefik.yml

```diff
@@ -0,0 +1,68 @@
+version: '3.7'
+
+services:
+  users:
```

Comment:
As we control the DNS on the server-side, there is no problem in routing requests to `users, things, authn` services.